nqeit-presto-extension/build.gradle.kts

```diff
@@ -29,6 +29,8 @@ dependencies {
   compileOnly(libs.microprofile.openapi)
   compileOnly(libs.jetbrains.annotations)
 
+  compileOnly(libs.guava)
```

Comment:
needed because of:
```
> Task :nqeit-iceberg-flink-extension:compileJava
/home/runner/work/query-engine-integration-tests/query-engine-integration-tests/nqeit-iceberg-flink-extension/src/main/java/org/projectnessie/integtests/flink/IcebergFlinkExtension.java:21: error: package com.google.common.collect does not exist
import com.google.common.collect.Lists;
```
i.e. we are using guava but not declaring a dependency (i guess it used to be pulled in by old flink version)